source/direct_debit/index.html.md.erb

```diff
@@ -255,96 +133,197 @@ Required arguments:
  "amount"
  "description"
  "reference"
- "agreement_id"
- }
- ```
+ "mandate_id"
+}
+```
 
 Once a mandate has been set up and a mandate ID is available, you can make an
 API call to collect a payment against the mandate.
 
-You do not need to wait until the mandate is confirmed to send a collection
-request to GOV.UK Pay.
+You do not need to wait until the mandate is confirmed to send a collection request to GOV.UK Pay.
```

Comment:
But you do need to wait until the user has entered their details (`pending`), which is probably worth highlighting here.

Reply:
I've clarified this in https://github.com/alphagov/pay-tech-docs/pull/298/commits/562c05d6cc099a67190a4bfe298a6acc6291794c.